Makefile

```diff
@@ -149,6 +149,10 @@ bench: node_modules $(GEN)/protobuf-bench $(BUILD)/protobuf ## Benchmark code si
 perf: $(BUILD)/protobuf-test
 	npm run -w packages/protobuf-test perf
 
+.PHONY: perfnext
+perfnext: $(BUILD)/protobuf-test
+	npm run -w packages/protobuf-test perf:next
+
```

Comment:
We will remove v1 today. Let's replace the old make target, so we don't have to touch this twice.